webapp/src/components/topBar.tsx

```diff
@@ -4,16 +4,28 @@
 import React from 'react'
 
 import './topBar.scss'
+import {FormattedMessage} from 'react-intl'
+
 import HelpIcon from '../widgets/icons/help'
 import {Utils} from '../utils'
 import {Constants} from '../constants'
 
 const TopBar = React.memo((): JSX.Element => {
     if (Utils.isFocalboardPlugin()) {
+        const feedbackUrl = 'https://www.focalboard.com/fwlink/feedback-boards.html?v=' + Constants.versionString
         return (
             <div
                 className='TopBar'
             >
+                <a
+                    className='link'
+                    href={feedbackUrl}
+                >
+                    <FormattedMessage
+                        id='TopBar.give-feedback'
+                        defaultMessage='Give Feedback'
+                    />
+                </a>
```

Comment:
Need to run `npm run i18n-extract` in webapp to update language files. Unless there are already there.

Reply:
Thanks, good catch. Ran and updated.